.github/actions/setup-adb/action.yaml

```
@@ -0,0 +1,46 @@
+  name: Create Android Instance and Connect ADB
+
+  inputs:
+    android-version:
+      default: 15
+      description: Set jammy:android version to use
```

Is this description up-to-date? From the name of the input, I would expect it to be only the android version.

---

@ajanon  yes it is only the version number as in " jammy:android${ANDROID_VERSION}" .. Do you have in mind a different format ?

---

Ah, yes, I just feels liks it's not necessary to explain the format here.
I think something like the following could be sufficient:

```suggestion
      description: Set android version to use
```

Wdyt?

Also, we may want to be able to add the Ubuntu version as an additional input. For now, we only support jammy, but this may (will) change in the future.

---

That's a good idea, would also give flexibility to run a matrix with multiple versions  & images

---

Done